datadog_sync/commands/_import.py

```diff
@@ -6,12 +6,13 @@
 from click import command
 
 from datadog_sync.constants import Command
-from datadog_sync.commands.shared.options import common_options, source_auth_options
+from datadog_sync.commands.shared.options import common_options, destination_auth_options, source_auth_options
 from datadog_sync.commands.shared.utils import run_cmd
 
 
 @command(Command.IMPORT.value, short_help="Import Datadog resources.")
 @source_auth_options
+@destination_auth_options
```

Comment:
Allows us to pass in destination url, api key, and app key to `import` command. We can use these to send metrics to source and destination. These aren't required so you can still run an `import` without them.